framework/src/meshgenerators/FileMeshGenerator.C

```diff
@@ -28,6 +28,10 @@ FileMeshGenerator::validParams()
   params.addParam<std::vector<std::string>>(
       "exodus_extra_element_integers",
       "The variable names in the mesh file for loading extra element integers");
+  params.addParam<bool>("use_for_exodus_restart",
+                        false,
+                        "True to indicate that the mesh file this generator is used can be used "
+                        "for restarting variables");
```

Comment:
This doc string is confusing. Maybe "True to indicate that the mesh file this generator is reading can be used for restarting variables"?

Reply:
Sure.